package.json

```diff
@@ -14,54 +14,46 @@
   "author": "",
   "license": "LGPL-3.0-only",
   "dependencies": {
-    "@ant-design/icons": "^4.7.0",
-    "@reduxjs/toolkit": "^1.8.1",
-    "@types/react": "^18.0.28",
-    "@types/react-dom": "^18.0.10",
-    "@types/recharts": "^1.8.24",
-    "antd": "^5.0.0",
-    "axios": "^0.24.0",
+    "@ant-design/icons": "^5.0.1",
+    "@reduxjs/toolkit": "^1.9.3",
+    "antd": "^5.2.2",
+    "axios": "^1.3.4",
     "bento-charts": "^1.0.7",
```

Comment:
Bento-charts changes from [this](https://github.com/bento-platform/bento_public/pull/64) pr need to be double checked. Error occuring in bento_public when displaying data charts in overview. Very likely due to React dependency errors

Problem:
- The bento-charts [repo](https://github.com/bento-platform/Bento-Charts) has to package.json
- Oddly, v1.0.7 was published with a package.json that includes conflicting dependencies with this branch of bento_public (react especially)
- Bento-charts should specify [peerDependencies](https://nodejs.org/ko/blog/npm/peer-dependencies) instead of dependencies for compatibility

Reply:
Ongoing PR [here](https://github.com/bento-platform/Bento-Charts/pull/1) .
Waiting for review and NPM_TOKEN to be added to the Actions secrets for CI publish.

Reply:
Testing with new package 1.0.8 before creating 2.0.0

Reply:
Updated to 2.0.0 release.